README.md

```diff
@@ -69,20 +90,27 @@ mkdir -p ~/catkin_ws/src
 apt install ros-kinetic-mavros ros-kinetic-mavros-extras
 ```
 
-6. Install avoidance module dependencies (pointcloud library and octomap).
+6. Install the geographiclib dataset
+
+```bash
+wget https://raw.githubusercontent.com/mavlink/mavros/master/mavros/scripts/install_geographiclib_datasets.sh
+sudo ./install_geographiclib_datasets.sh
+```
+
+7. Install avoidance module dependencies (pointcloud library and octomap).
 
 ```bash
```

Comment:
FYI, if you indent 3 spaces to the bullet then code gets indented under the bullet.